.storybook/main.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   stories: ['../src/**/*.stories.@(js|jsx|ts|tsx|mdx)'],
+  staticDirs: ['../public'],
```

Comment:
Setting the static directory in `package.json` was deprecated, it has to be set here instead.